connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2238,6 +2238,1359 @@ object functions {
    */
   def radians(columnName: String): Column = radians(Column(columnName))
 
+  //////////////////////////////////////////////////////////////////////////////////////////////
+  // Misc functions
+  //////////////////////////////////////////////////////////////////////////////////////////////
+
+  /**
+   * Calculates the MD5 digest of a binary column and returns the value as a 32 character hex
+   * string.
+   *
+   * @group misc_funcs
+   * @since 1.5.0
```

Comment:
Ur, I'm a little confused. Is it correct, @hvanhovell and @zhengruifeng ?

cc @HyukjinKwon

Reply:
yeah let's change `since`